scapy/layers/dcerpc.py

```diff
@@ -300,13 +301,17 @@ class NetlogonAuthSignature(Packet):
             },
         ),
         XLEShortField("Pad", 0xFFFF),
-        ShortField("Reserved", 0),
-        XLELongField("SequenceNumber", 0),
+        ShortField("Flags", 0),
+        XStrFixedLenField("SequenceNumber", b"", length=8),
```

Comment:
What's the reason for changing this type? The name seems to indicate the prior type was more appropriate.

Reply:
I thought like you but look at this mess
![image](https://user-images.githubusercontent.com/10530980/194141275-306aa93d-65ae-42ea-8280-8d3c67d76b32.png)
and
![image](https://user-images.githubusercontent.com/10530980/194141477-c610b01f-f940-46c7-a37d-250e7ff627ff.png)

So it's not actually a number on the wire :/